lib/routes/cags/zs-sss.ts

```diff
@@ -0,0 +1,68 @@
+import ofetch from '@/utils/ofetch';
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/zs-sss',
+    categories: ['university'],
+    example: '/cags/zs-sss',
+    features: {
+        antiCrawler: false,
+        requireConfig: false,
+        requirePuppeteer: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    maintainers: ['Chikit'],
+    name: '中国地质科学院研究生院硕士生招生信息',
+    parameters: {
+        no_parameters: '该路由无需额外参数',
+    },
+    radar: [
+        {
+            source: ['edu.cags.ac.cn/#/dky/list/barId=zs_sss/cmsNavType=1'],
+            target: '/zs-sss',
+        },
+    ],
+    handler: async () => {
+        const API_URL = 'https://edu.cags.ac.cn/api/cms/cmsNews/pageByCmsNavBarId/zs_sss/1/10/0';
+        const MAX_INTRO_LENGTH = 200;
+
+        const response = await ofetch(API_URL);
+        const data = response.data;
+
+        const items = data.map((item) => {
+            const id = item.id;
+            const title = item.title;
+
+            let introduction = item.introduction.replaceAll(/\s+/g, ' ').trim();
+            if (introduction.length > MAX_INTRO_LENGTH) {
+                introduction = `${introduction.substring(0, MAX_INTRO_LENGTH)}...`;
+            }
```

Comment:
Don't trim the content yourself. 
RSSHub will handle it https://github.com/DIYgod/RSSHub/blob/fe22d59de6272079bd37f128d60bdcb4a520d523/lib/middleware/parameter.ts#L400-L414